configs/setups/awicm3/awicm3.yaml

```diff
@@ -323,11 +378,11 @@ fesom:
 
         coupling_freq_in_steps: $((${oasis3mct.coupling_time_step} / ${fesom.time_step}))
 
-        add_config_files:
-                cvmix: cvmix
+        #add_config_files:
+                #cvmix: cvmix
 
-        add_config_sources:
-                cvmix: "${namelist_dir}/namelist.cvmix"
+        #add_config_sources:
+                #cvmix: "${namelist_dir}/namelist.cvmix"
```

Comment:
Doesn't other versions of AWICM3 need the `cvmix` namelist?

Reply:
not by default

Reply:
Wouldn't it be cleaner to add this on the `fesom.yaml` instead?

Reply:
I'm keeping my modifications to the awicm3.yaml mostly. Since I'm basing off of fesom2.0.yaml there is no cvmix in there. I think it works ok like this.